utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\Packagist\PackageProvider;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * Packages that are not released - old packages or forks
+     * @var string[]
+     */
+    private const EXCLUDED_PACKAGES = [
+        // excluded from the initial tagging as there needs to be another commit with composer.lock and package-lock.json
+        // @see https://github.com/shopsys/shopsys/pull/1264
+        'shopsys/shopsys',
+        'shopsys/project-base',
+        // not maintained anymore
+        'shopsys/product-feed-interface',
+        'shopsys/phpstorm-inspect',
+        // forks
+        'shopsys/postgres-search-bundle',
+        'shopsys/doctrine-orm',
+        'shopsys/jparser',
+        // old packages
+        'shopsys/syscart',
+        'shopsys/sysconfig',
+        'shopsys/sysreports',
+        'shopsys/sysstdlib',
+    ];
+
+    /**
+     * @var \Shopsys\Releaser\Packagist\PackageProvider
+     */
+    private $packageProvider;
+
+    /**
+     * @param \Shopsys\Releaser\Packagist\PackageProvider $packageProvider
+     */
+    public function __construct(PackageProvider $packageProvider)
+    {
+        $this->packageProvider = $packageProvider;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and push git tags for packages excluding monorepo and project-base';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 640;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $packages = $this->packageProvider->getPackagesByOrganization('shopsys', self::EXCLUDED_PACKAGES);
+        $packages = str_replace('shopsys/', '', $packages);
+
+        $versionString = $version->getVersionString();
+
+        $this->processRunner->run('mkdir temp');
+        $packageNamesWithProblems = [];
+        foreach ($packages as $packageName) {
+            $this->processRunner->run('cd temp && git clone https://github.com/shopsys/' . $packageName . '.git');
```

Comment:
This line takes a lot of time sometimes, it would be nice to have a progressbar of the `git clone` here. If not progress bar then at least a note like "Clonning shopsys/xxx, this can take couple of minutes" and annimation that would tell me the process is working. As an animation it would be nice to have dots `.`->`..`->`...`->`.`->`..`->and again, and again....

Reply:
I added a note